ursa.ino

```diff
@@ -20,6 +20,22 @@ rmt_config_t configL;//settings for RMT pulse for stepper motor
 rmt_item32_t itemsL[1];//holds definition of pulse for stepper motor
 rmt_config_t configR;
 rmt_item32_t itemsR[1];
+byte voltage = 0; //0v=0 13v=255
+int signalStrength = -100;
+byte wifI = 0; //counter for wifi sending and recieving

```

Comment:
why is the I capitalized?

Reply:
It's emphasizing individuality in modern society

Reply:
because it is a counter for wifi and i is often used as a counter, so I emphasized the I in wifI and I think it's really funny

Reply:
Oof. Humor is often difficult in code because it can _reallllly_ take away from the meaning and add lots of confusion (as it did here).

Reply:
Good point. do you have a recommendation for what to call the variable?

Reply:
I'm still a bit confused by what it's used for. From the comment, I'd say maybe `throughput` or `bytesSentAndReceived` or `bytesCommunicated`?

Reply:
Actually, going through the code a little more, maybe more like `transactionCount` or `activityCount`

Reply:
I think bytesCommunicated is close. It is a counter used for stepping through and taking data out of one array, and then it is reset and used for stepping through and putting data in another array

Reply:
@connor fixed this beautifully